src/main/java/github/scarsz/discordsrv/objects/managers/link/JdbcAccountLinkManager.java

```diff
@@ -79,16 +80,22 @@ public static boolean shouldUseJdbc(boolean quiet) {
         if (StringUtils.isBlank(jdbc)) return false;
 
         Matcher matcher = JDBC_PATTERN.matcher(jdbc);
-        if (!matcher.find() || matcher.groupCount() < 4) {
-            if (!quiet) DiscordSRV.error("Not using JDBC because < 4 matches for JDBC url");
+        
+        if (!matcher.matches()) {
+            if (!quiet) DiscordSRV.error("connection string '" + jdbc + "' is NOT valid!");
+            return false;
+        }
+        
+        if (matcher.group("DBEngine").equalsIgnoreCase("DBCC")) {
```

Comment:
Messed up when you tried manually editing Vankka's previous suggestion.
```suggestion
        if (!matcher.group("DBEngine").equalsIgnoreCase("jdbc")) {
```